### ISSUE TITLE
[HELP] Cannot start K3s Server

### ISSUE THREAD

**Issue by dunmorogher** [k3s-io/k3s#10618]
K3s Version: 
```
k3s version v1.30.2+k3s1 (aa4794b3)
go version go1.22.4
```

Node(s) CPU architecture, OS, and Version: 
`Linux k8s-master 5.14.0-427.26.1.el9_4.x86_64 #1 SMP PREEMPT_DYNAMIC Wed Jul 17 15:51:13 EDT 2024 x86_64 x86_64 x86_64 GNU/Linux`

Cluster Configuration: N/A

**Describe the bug:**
I downloaded the packages `k3s`, `k3s-airgap-images-amd64.tar.gz`, `install.sh` at https://github.com/k3s-io/k3s/releases/tag/v1.30.2%2Bk3s1 to install K3s locally.

1. `mv k3s /usr/local/bin; chmod +x /usr/local/bin/k3s`
2. `mkdir -p /var/lib/rancher/k3s/agent/images/; cp ./k3s-airgap-images-amd64.tar.gz /var/lib/rancher/k3s/agent/images/`
3. `INSTALL_K3S_SKIP_DOWNLOAD=true ./install.sh`

After the installation, I used `kubectl get nodes` to get nodes; however, I failed to get it as below:

![image](https://github.com/user-attachments/assets/192fc668-2101-44e6-826e-ce820840e90f)

I also downloaded the log with `journalctl -u k3s > k3s.log` and I found several errors in the log; attached the log here...

[k3s.log](https://github.com/user-attachments/files/16444618/k3s.log)

**Steps To Reproduce:**
1. I disabled the firewall and allow all traffic in iptables
    ```shell
    systemctl disable --now firewalld
    iptables -F; iptables -t nat -F; iptables -t mangle -F; iptables -X
    iptables -P INPUT ACCEPT; iptables -P FORWARD ACCEPT; iptables -P OUTPUT ACCEPT
    ```
2. Disabled SELinux

    ![image](https://github.com/user-attachments/assets/785c0a5e-4799-4386-966d-6a7b124254fc)

3. Did the steps to install K3s offline as stated in the section above.

**Comment by dereknola** [comment]
What do the actualy logs say `journalctl -u k3s`?

What does your K3s config look like `cat /etc/rancher/k3s/config.yaml`?

Are you using a seperately installed kubectl, or the one that comes with K3s (i.e. make sure your KUBECONFIG is pointing at `/etc/rancher/k3s/k3s.yaml`?

**Comment by brandond** [comment]
The logs seem to stop partway through normal startup, however I don't see any errors.

> After the installation, I used kubectl get nodes to get nodes; however, I failed to get it as below:

Are you sure you're using the k3s kubeconfig? K3s isn't listening on `127.0.0.1:43397`. Make sure you don't have the KUBECONFIG environment variable set and pointing at some other kubeconfig. You should be using the kubeconfig at /etc/rancher/k3s/k3s.yaml. If you're using the kubectl that comes with k3s, it will use this file by default.

**Comment by dunmorogher** [comment]
Hi @dereknola, @brandond - thank you for reply! 

After the error's happening, I uninstalled the whole K3s. As you all mentioned `kubectl` - I found out I did not use the default `kubectl`...instead, I used `kubectl` installed by `dnf install -y kubectl` by my own.
I removed the `kubectl` and directly executed the command as described in K3s website - `curl -sfL https://rancher-mirror.rancher.cn/k3s/k3s-install.sh | INSTALL_K3S_MIRROR=cn sh -` (the version of K3s seems not up to date; it is 1.29.x), finally K3s started well! :-) 

I will take a look at the documentation for knowing more.
